Dockerfile

```diff
@@ -20,15 +20,22 @@ RUN yum update -y \
         cyrus-sasl-devel \
         libxml2-devel \
         libxslt-devel \
-        nodejs \
         zlib-devel \
-        mysql-community-libs-compat-5.6.14-3.el6.x86_64 \
+        git \
```

Comment:
For verything that we're aquiring from github in https://github.com/mozilla/olympia/blob/master/requirements/prod.txt#L95